### PR TITLE
chore: add e2e tests for organization members

### PR DESCRIPTION
**Pull request by aslilac**

### CODE REVIEW

File: site/e2e/helpers.ts [hunk 1]
```diff
@@ -987,3 +987,48 @@ export async function openTerminalWindow(
...
+	await page.getByRole("option", { name: "Password", exact: false }).click();
+	// Using input[name=password] due to the select element utilizing 'password'
+	// as the label for the currently active option.
+	const passwordField = page.locator("input[name=password]");
+	await passwordField.fill(password);
+	await page.getByRole("button", { name: "Create user" }).click();
+	await expect(page.getByText("Successfully created user.")).toBeVisible();
+
+	await expect(page).toHaveTitle("Users - Coder");
+	await expect(page.locator("tr", { hasText: email })).toBeVisible();
```
Comment (aslilac) [inline site/e2e/helpers.ts:1030]: This is all taken verbatim from the existing `createUserWithPassword` test. It just seems like the kind of thing that's helpful to turn into a reusable function.

**Review by jaaydenh — APPROVED** [review]
Overall looks good. How about adding a test to verify a user cannot edit their own roles?

Also, in the future once this PR and the custom roles e2e PR is merged. I think it could be useful to create a custom role and test assigning the custom role to a user.

**Comment by aslilac** [comment]
Yeah, custom roles have their own ticket for adding tests. Those are next.

**Comment by aslilac** [comment]
Closes coder/internal#165

**Pull request merged**